samplesheets/vueapp/src/components/Overview.vue

```diff
@@ -33,15 +33,16 @@
         <dl class="row pb-0">
           <dt class="col-md-3">iRODS Repository</dt>
           <dd class="col-md-9">
-            <span v-if="sodarContext.irods_status"
-                  class="badge badge-pill badge-success">
-              Available
+            <span v-if="sodarContext.irods_status">
```

Comment:
Unfortunately this doesn't work: the badge remains in the "Updating.." phase forever, see accompanying screenshot.

No ajax request is received by the server either and upon closer inspection, it seems the `updateStats()` method is never called. See `SheetTableHeader` for an example on how to do this.

Also, as with other features, make sure to verify in your dev environment that this works as expected before sumitting the PR for review :)

![2023-07_inv_badge_bug](https://github.com/bihealth/sodar-server/assets/11211145/ac7725af-b255-4056-9faf-bb07b5bd639c)

Reply:
This works now, good work fixing the issue.